config.yaml

```diff
@@ -59,6 +59,27 @@ contracts:
     events:
       - event: NotifyReward(address indexed from, uint256 amount)
 networks:
+  - id: 34443 # Mode
+    start_block: 0
```

Comment:
I personally prefer including the start block -  But it does probably only save max 3 seconds or so of indexing time.